### ISSUE TITLE
Provide a centralized Model_Instance_ID-to-Model_Name Map

### ISSUE THREAD

**Issue by liangfok** [RobotLocomotion/drake#3053]
The Model_Instance_Name-to-Model_Name map should probably be stored in the `RigidBodyTree`. Having such a map will eliminate the need to store the model name in every modeling element like `RigidBody` and `RigidBodySensor`.

**Comment by RussTedrake** [comment]
Absolutely agree.  We should also modify the lcm_viewer_load_robot message to send over those model names, so that it can be displayed properly in the scene graph.  discussed quickly w/ @patmarion

**Comment by jwnimmer-tri** [comment]
Moving over to @sammy-tri per recent work like #8928's second incarnation.  I suspect you can close this either now or once the second incarnation lands, perhaps?

**Comment by sammy-tri** [comment]
Closing because (1) done on `MultibodyTree` and (2) WONTFIX on `RigidBodyTree`.
